### PR TITLE
profile followers endpoint

### PR DESCRIPTION
**Pull request by ahsu1230**
@2is10

### CODE REVIEW

**Comment by ahsu1230** [comment]
just saw the change for `loadFullConnectionsUser()` - change accordingly! =D

File: kifi-backend/modules/shoebox/app/com/keepit/controllers/website/UserController.scala [hunk 1]
```diff
@@ -77,7 +77,7 @@ class UserController @Inject() (
       //global or mutual
       val libCount = viewer.map(u => libraryRepo.countLibrariesForOtherUser(ownerId, u)).getOrElse(libraryRepo.countLibrariesOfUserFromAnonymous(ownerId)) //not cached
       //global
-      val followersCount = libraryMembershipRepo.countFollowersWithOwnerId(ownerId) //cached
+      val followersCount = libraryMembershipRepo.countFollowersForOwner(ownerId) //cached
```
Comment (2is10) [inline kifi-backend/modules/shoebox/app/com/keepit/controllers/website/UserController.scala]: Which method we use here should depend on who the viewer is, no?

Comment (ahsu1230) [inline kifi-backend/modules/shoebox/app/com/keepit/controllers/website/UserController.scala]: yup you're right! was just about to fix that!

File: kifi-backend/modules/shoebox/app/com/keepit/controllers/website/UserController.scala [hunk 2]
```diff
@@ -135,8 +135,8 @@ class UserController @Inject() (
         log.warn(s"can't find username ${username.value}")
         Future.successful(NotFound(s"username ${username.value}"))
       case Some(user) =>
-        val viewerId = request.userIdOpt.getOrElse(user.id.get)
-        userConnectionsCommander.getConnectionsSortedByRelationship(viewerId, user.id.get) map { connections =>
+        val viewerIdOpt = request.userIdOpt.getOrElse(user.id.get)
+        userConnectionsCommander.getConnectionsSortedByRelationship(viewerIdOpt, user.id.get) map { connections =>
```
Comment (2is10) [inline kifi-backend/modules/shoebox/app/com/keepit/controllers/website/UserController.scala:148]: The value you’ve renamed is not an `Option`. Perhaps you meant:

``` scala
val viewerIdOpt = request.userIdOpt
userConnectionsCommander.getConn...(viewerIdOpt.orElse(user.id), user.id.get) ...
```

**Comment by ahsu1230** [comment]
uhoh looks like I messed up somewhere with this branch

**Comment by 2is10** [comment]
Eishay just committed a cache that may be helpful in #14069.

**Pull request closed without merging**